open-api/rest-catalog-open-api.yaml

```diff
@@ -2800,6 +3062,58 @@ components:
           additionalProperties:
             type: string
 
+    PrePlanTableRequest:
+      type: object
+      required:
+        - select
+        - options
+      properties:
+        select:
+          description: A list of the selected columns that user specifies during a query.
+          type: array
+          items:
+            type: string
+        filter:
+          $ref: '#/components/schemas/Expression'
+        options:
+          type: object
+          additionalProperties:
+            type: string
+        snapshot-id:
```

Comment:
missing from-snapshot-id and to-snapshot-id for incremental read support.

Reply:
Just want to confirm this params apply for both `preplan` and `plan` requests right?

Reply:
yes